Formula/rom-tools.rb

```diff
@@ -30,7 +29,9 @@ def install
     # Cut sdl2-config's invalid option.
     inreplace "scripts/src/osd/sdl.lua", "--static", ""
 
-    system "make", "TOOLS=1",
+    # Use bundled asio instead of latest version.
```

Comment:
This should use the brewed version

Reply:
It can't.
See #49695.

Reply:
Can you link https://github.com/mamedev/mame/issues/5721 then

Reply:
Then what? Add comments in commit?

Reply:
Make the comment reference the issue